.circleci/config.yml

```diff
@@ -49,6 +50,12 @@ jobs:
           command: |
             . venv/bin/activate
             python test.py
+
+      - run:
+          name: Code Style Check
+          command: |
+            . venv/bin/activate
+            flake8 vibora
```

Comment:
This misses the Python syntax error in #32 that flake8 is capable of finding.

Reply:
all right，I will recovery it

Reply:
@cclauss 
I check this issue. The author has fixed this problem.